src/Xamarin.Android.Build.Tasks/Tasks/RuntimeIdentifierToAbi.cs

```diff
@@ -14,17 +14,21 @@ public class RuntimeIdentifierToAbi : AndroidTask
 
 		public string RuntimeIdentifier { get; set; }
 
+		public string [] RuntimeIdentifiers { get; set; }
+
 		[Output]
 		public string SupportedAbis { get; set; }
 
 		public override bool RunTask ()
 		{
-			if (!string.IsNullOrEmpty (RuntimeIdentifier)) {
+			if (RuntimeIdentifiers != null && RuntimeIdentifiers.Length > 0) {
+				SupportedAbis = string.Join (";", RuntimeIdentifiers.Select (MonoAndroidHelper.RuntimeIdentifierToAbi));
+			} else if (!string.IsNullOrEmpty (RuntimeIdentifier)) {
 				SupportedAbis = MonoAndroidHelper.RuntimeIdentifierToAbi (RuntimeIdentifier);
 			}
 			if (string.IsNullOrEmpty (SupportedAbis)) {
```

Comment:
Different question: why isn't this a dead code path?  How could we possibly get here, given that *both* `$(AndroidSupportedAbis)` and `$(RuntimeIdentifiers)` provide default values if they're initially empty?

Though I suppose if someone explicitly overrides on the command-line to be empty -- is that possible? -- that might trigger this codepath?

    msbuild /p:AndroidSupportedAbis= /p:RuntimeIdentifiers=

though that feels like an "unkind" move, and perhaps *deserves* a build failure.

Reply:
It would get here if you provided `RuntimeIdentifiers=somethinginvalid`, so maybe this just needs to be an error instead of silently using some default.

Reply:
I added `XA0035`, which seems a lot better.